dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -297,6 +298,56 @@ public Organization updateOrganizationDescription(@ApiParam(hidden = true) @Auth
         return eventDAO.findEventsForOrganization(id, offset, limit);
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{organizationId}/star")
+    @ApiOperation(value = "Star an organization.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) })
+    public void starOrganization(@ApiParam(hidden = true) @Auth User user,
```

Comment:
Ok for now but should be using OpenApi3 for new endpoints (can do a combination of both too).